dali/operators/image/convolution/gaussian_blur.cc

```diff
@@ -97,7 +96,7 @@ class GaussianBlurOpCpu : public OpImplBase<CPUBackend> {
 
     output_desc.resize(1);
     output_desc[0].type = type2id<Out>::value;
-    output_desc[0].shape.resize(nsamples, input.shape().sample_dim());
+    // Shape is set by ProcessOutputDesc
```

Comment:
Hmm... this breaks the promise of being a drop-in extension to sequence op. Can't ProcessOutputDesc reconstitute outputs from per-frame output descs?

Reply:
As we discussed offline, it's just optimization: not to create tensorlists back and forth and to avoid validation of shapes consistency. But it would work just fine it we set the shape here.